Source/System.Management/Pash/ParserIntrinsics/AstBuilder.cs

```diff
@@ -424,11 +424,20 @@ private StatementAst BuildBreakStatementAst(ParseTreeNode parseTreeNode)
 
         StatementAst BuildTryStatementAst(ParseTreeNode parseTreeNode)
         {
+            var subClauses = parseTreeNode.ChildNodes[0].ChildNodes;
+            var catchClauses = subClauses[2];
```

Comment:
Won't this be incorrect on multiple `catch` blocks?

Ref: http://stackoverflow.com/questions/8381237/powershell-catching-typed-exceptions#8381798